src/runner/observer.rs

```diff
@@ -16,3 +16,18 @@ pub trait RunnerObserver: Send + Sync {
     fn enter_example(&self, example: &ExampleHeader);
     fn exit_example(&self, example: &ExampleHeader, report: &ExampleReport);
 }
+
+#[cfg(feature = "specialization")]
```

Comment:
that was not what I meant. You can make a default impl for a trait like that:
```rust
trait Hello {
  fn default_int(&self) -> u32 { 24 }
}
struct A;
struct B;

impl Hello for A {}
impl Hello for B {
  fn default_int(&self) -> u32 { 42 }
}
```
It's standard and stable :)

Reply:
Fixed.